app/code/Magento/Widget/Model/Widget/Instance.php

```diff
@@ -30,7 +30,7 @@ class Instance extends \Magento\Framework\Model\AbstractModel
 
     const PRODUCT_LAYOUT_HANDLE = 'catalog_product_view';
 
-    const SINGLE_PRODUCT_LAYOUT_HANLDE = 'catalog_product_view_id_{{ID}}';
+    const SINGLE_PRODUCT_LAYOUT_HANDLE = 'catalog_product_view_id_{{ID}}';
```

Comment:
Constant removing is backward incompatible, please, fix issue according to [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.1/contributor-guide/backward-compatible-development/index.html)

Reply:
Hi @sidolov 
Do you mean to mark the constant with typo deprecated and not delete it? Then just declare correct one.

Reply:
Yes, you are right